### ISSUE TITLE
Where to download the PixelVisionOS?

### ISSUE THREAD

**Issue by andy3520** [PixelVision8/PixelVisionOS#195]
I'm confused, I follow the instruction here: https://github.com/PixelVision8/PixelVision8/wiki/quick-start to install the PV8 and it said I need PixelVisionOS and link to here, but this is an empty repo? so where should I look to get the OS? and the example repo is also empty?

**Comment by velteyn** [comment]
Yes, also I have this question.. Is it on the PRO version only ?  Cannot find Pro version either ...

**Comment by Kaushikpadmanabhan** [comment]
> Yes, also I have this question.. Is it on the PRO version only ? Cannot find Pro version either ...

Its open source no, so there is no Pro version anymore. But am confused on where to OS went...

**Comment by xelfe** [comment]
Any news about it ? I would like to use the PixelVisionOs on latest version too...

**Comment by Azurantine** [comment]
Just noticed the latest version is newer than what I have installed, so went to update to this version only to discover the issue here - the PV-OS is no longer available to download. Any updates on this yet?

**Comment by Kaushikpadmanabhan** [comment]
For anyone wondering. When I asked on the discord. They mentioned it has been taken off to be updated to go with the newest version of pixel vision and should be back in a week or so

**Comment by Azurantine** [comment]
> For anyone wondering. When I asked on the discord. They mentioned it has been taken off to be updated to go with the newest version of pixel vision and should be back in a week or so

That explains it then, so it wasn't Jesse trying to send us all mad then at least.

**Comment by BrunoDSL** [comment]
So...How many months long is that "week or so?"

**Comment by velteyn** [comment]
I think even a broken version could be useful: because we can help fix it. He can release it and open an issue where he asks for help.. No ?
